packaging/config/functions.go

```diff
@@ -27,3 +27,18 @@ func GetCloudArchiveSource(series string) (packaging.PackageSource, packaging.Pa
 		return configureCloudArchiveSourceUbuntu(series)
 	}
 }
+
+func RequiresBackports(series string, pkg string) bool {
+	backportPkgs, ok := backportsBySeries[series]
```

Comment:
This `ok` check is unnecessary, you'll just range over a nil slice below and then return false anyway.